src/model/entities/segment.ts

```diff
@@ -241,4 +241,8 @@ export class Segment {
   public getUnsyncedCopy(): Segment {
     return Object.assign(Object.create(Object.getPrototypeOf(this)), this, { id: `${this.id}${UNSYNCED_ID_POSTFIX}`})
   }
+
+  public getIsSegmentUntimed(): boolean {
+    return !this.getParts().some(part => !part.isUntimed())
```

Comment:
So if a segment contains one or more parts that are timed, then the segment is untimed?

Are you sure the negations are suppose to be here?

Reply:
If `this.getParts().some(part => !part.isUntimed())` returns true, that means that atleast one Part is `timed`.
If atleast one Part is `timed`, then the Segment is also `timed` so in order for `getIsSegmentUntimed()` to return true when it's `untimed`, it needs to negate `this.getParts().some(part => !part.isUntimed())`.

So yes, the negations are needed.

Reply:
This says that if a segment has at least one part that is timed. The segment is considered timed, and the method will therefore return `false`.
If we have no timed parts in the segment, it will return `true`, meaning the segment is untimed.